src/custom/state/operator/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAction } from '@reduxjs/toolkit'
+
+// modeled after state/user/actions
+export interface SerializedToken {
```

Comment:
Probably not needed if you use the address as David and u commented
Also, I don't love the word "serialize". I think we should model the state as we want, not conditionally to knowing we run a middleware that has some restrictions.

Actually, I'm not sure if they have these restrictions. I kind of asked that in other comment, so I'm not open it again here.